server/src/internalClusterTest/java/org/opensearch/indices/stats/IndexStatsIT.java

```diff
@@ -183,6 +183,7 @@ public void testFieldDataStats() throws InterruptedException {
         client().prepareIndex("test").setId("2").setSource("field", "value2", "field2", "value2").execute().actionGet();
         client().admin().indices().prepareRefresh().execute().actionGet();
```

Comment:
Should we refactor all these direct invocations of refresh to use the `refresh()` helper method in the base class?

Reply:
Yes that's good idea. I have refactored direct `client().admin().indices().prepareRefresh().execute().actionGet();` call to `refresh()` helper method